cmd/spiffe/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	spiffe "github.com/spiffe/go-spiffe"
+)
+
+func main() {
+	// Make log not print out time information
+	log.SetFlags(0)
+
+	if len(os.Args) <= 1 {
+		log.Fatalf("\033[31mexpecting path [paths...] e.g\033[00m\n\tspiffe /var/lib/letsencrypt/spiffe.pem $HOME/Desktop/certs/cert.pem\033[00m\n")
+	}
+
+	seenPaths := make(map[string]bool)
```

Comment:
Good question @evan2645, am examining multiple certs because that creates a nice use case for the CLI, that way when testing, one can supply multiple certs on the CLI which is easier to do than writing a unit test.